src/routes/Project/components/Table.js

```diff
@@ -215,10 +226,13 @@ export default class Table extends Component {
       </div>
     );
   }
+
```

Comment:
http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline